src/searchForm/index.tsx

```diff
@@ -84,9 +80,15 @@ const StyledSearchForm = styled(SearchForm)`
     box-sizing: border-box;
     line-height: ${font.l.lineHeight};
     width: 100%;
+    min-width: 0;
+    text-align: left;
     white-space: nowrap;
     overflow: hidden;
     align-items: center;
+
+    & > *:not(:first-child) {
+      margin-left: ${space.m};
```

Comment:
Why do we need this?

Reply:
This ensures that child elements of a `.kirk-search-button` are spaced by `space.m`.
With this selector we no longer need:

```css
// line 63
& .kirk-bullet--searchForm {
  margin-right: ${space.m};
}

// line 132
& .kirk-search-button svg {
  margin-right: ${space.m};
}
```

And it fixes the issue with small screens of the rotated invert icon's margin.